### PR TITLE
Issue #ED-2112 Notifications -> from SB-notification repo and Portal -> angular material UI integration

### PR DESCRIPTION
**Pull request by sri-pusuluri**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -39,5 +39,8 @@
       }
     }
   },
-  "defaultProject": "notification"
+  "defaultProject": "notification",
+  "cli": {
+    "analytics": "6b63c3ac-e480-4085-8624-e6fb2554641d"
```
Comment (rajeshkumaravel) [inline angular.json]: @sri-pusuluri please revert this change
CC: @rajeevsathish

File: projects/notification/tsconfig.lib.json [hunk 2]
```diff
@@ -23,7 +23,8 @@
     "fullTemplateTypeCheck": true,
     "strictInjectionParameters": true,
     "enableResourceInlining": true,
-    "compilationMode": "partial"
+    "compilationMode": "partial",
+    "enableIvy": "partial"
```
Comment (rajeshkumaravel) [inline projects/notification/tsconfig.lib.json]: @sri-pusuluri 
please enable ivy `enableIvy: true`
CC: @rajeevsathish

File: projects/notification/tsconfig.lib.prod.json [hunk 3]
```diff
@@ -4,6 +4,7 @@
     "declarationMap": false
   },
   "angularCompilerOptions": {
-    "compilationMode": "partial"
+    "compilationMode": "partial",
+    "enableIvy": "partial"
```
Comment (rajeshkumaravel) [inline projects/notification/tsconfig.lib.prod.json]: @sri-pusuluri
please enable ivy enableIvy: true
CC: @rajeevsathish

File: tsconfig.json [hunk 4]
```diff
@@ -1,7 +1,8 @@
 {
   "compileOnSave": false,
   "angularCompilerOptions": {
-    "compilationMode": "partial"
+    "compilationMode": "partial",
+    "enableIvy": "partial"
```
Comment (rajeshkumaravel) [inline tsconfig.json:5]: @sri-pusuluri
please enable ivy `enableIvy: true`
CC: @rajeevsathish

**Review by rajeshkumaravel — CHANGES_REQUESTED** [review]

File: projects/notification/tsconfig.lib.prod.json (hunk 3 above)
Comment (sri-pusuluri) [reply projects/notification/tsconfig.lib.prod.json]: done

File: angular.json (hunk 1 above)
Comment (sri-pusuluri) [reply angular.json]: updated

File: tsconfig.json (hunk 4 above)
Comment (sri-pusuluri) [reply tsconfig.json:5]: updated

**Review by rajeshkumaravel — APPROVED** [review]
